components/suggest/src/store.rs

```diff
@@ -337,16 +340,45 @@ where
                     .get_records_with_options(&options)?
                     .records;
 
-                self.ingest_records(writer, &records_chunk)?;
+                self.ingest_records(LAST_INGEST_META_UNPARSABLE, writer, &records_chunk)?;
```

Comment:
@linabutler I've been thinking about "unparsable" records a bit, and I see 2 options:

1. If we decide that we don't need to reprocess the record, then we can probably just get rid of all of the unparsable logic. It's a larger change, so might be better done in a separate PR. From my limited understanding of what unparsable records are meant for, I think it seems reasonable to simplify this to not have to reprocess unparsable records (because we won't be processing records of a type we don't understand with my change). For corrupted data, we will reprocessed the updated file, so we will unlikely need to reprocess an existing version.
2. If we decide to keep the logic for unparsable records (which is probably the safer approach), then we can keep the code that's kind of "duplicating" (i.e. use the exact record type like SuggestType::Icons.get_rs_type(), rather than pass in the last_ingest_key). That's because the match for the type will be done at the "match" statement to match the type of record. It's kind of annoying to pass the type multiple times, but it will prevent another matching done on this line to get the "actual" record type. 

Not sure if you have an opinion, but writing it out here so we can discuss this tomorrow.